src/sentry/options/defaults.py

```diff
@@ -2636,3 +2636,7 @@
     default=1,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "delayed_processing.batch_size",
+    default=10000,
+)
```

Comment:
You just need to add `flags=FLAG_AUTOMATOR_MODIFIABLE` and then you'll be good to go